src/megameklab/com/ui/Aero/Printing/PrintSmallCraftAerodyne.java

```diff
@@ -1,421 +1,422 @@
-        float shiftX = 7f;
-        float shiftY = 7f;
-        int pipsPerLine = 5;
-
-        Vector<float[]> pipPlotter = new Vector<float[]>(205, 1);
-
-        for (int lineCount = 1; lineCount <= 29; lineCount++) {
-            for (int point = 0; point < pipsPerLine; point++) {
-                pipPlotter.add(new float[]
-                    { pointX, pointY });
-                pointX += shiftX;
-            }
-
-            if (lineCount == 17) {
-                pipsPerLine += 5;
-                baseX -= shiftX * 5;
-            }
-
-            pointY += shiftY;
-            pointX = baseX;
-        }
-
-        ImageHelperAero.printArmorPoints(g2d, pipPlotter, totalArmor, 205);
-    }
-
-    private void printRightArmor(Graphics2D g2d, int totalArmor) {
-        float baseX = 434f;
-        float baseY = 237f;
-        float pointX = baseX;
-        float pointY = baseY;
-        float shiftX = 7f;
-        float shiftY = 7f;
-        int pipsPerLine = 5;
-
-        Vector<float[]> pipPlotter = new Vector<float[]>(205, 1);
-
-        for (int lineCount = 1; lineCount <= 29; lineCount++) {
-            for (int point = 0; point < pipsPerLine; point++) {
-                pipPlotter.add(new float[]
-                    { pointX, pointY });
-                pointX += shiftX;
-            }
-
-            if (lineCount == 17) {
-                pipsPerLine += 5;
-            }
-
-            pointY += shiftY;
-            pointX = baseX;
-        }
-
-        ImageHelperAero.printArmorPoints(g2d, pipPlotter, totalArmor, 205);
-
-    }
-
-    private void printStruct(Graphics2D g2d, int totalArmor) {
-        int posY = 369;
-        int posX = 269;
-        int[] topColumn = new int[]
-            { posY, posX };
-        int[] pipShift = new int[]
-            { 6, 6 };
-
-        Vector<int[]> pipPlotter = new Vector<int[]>(132);
-        for (int pos = 1; pos <= 32; pos++) {
-            pipPlotter.add(new int[]
-                { topColumn[0], topColumn[1] });
-            topColumn[0] += pipShift[0];
-            if ((pos % 8) == 0) {
-                topColumn[1] += pipShift[1];
-                pipShift[0] *= -1;
-                topColumn[0] += pipShift[0];
-            }
-        }
-
-        int pipSpace = (int) Math.floor(32 / totalArmor);
-        for (int pos = 0; pos < 32; pos += pipSpace) {
-            ImageHelperAero.drawAeroISPip(g2d, pipPlotter.get(pos)[0], pipPlotter.get(pos)[1]);
-            if (--totalArmor <= 0) {
-                return;
-            }
-        }
-    }
-
-    private void printSmallCraftAerodyneImage(Graphics2D g2d, Image img) {
-
-        int width = Math.min(220, img.getWidth(null));
-        int height = Math.min(130, img.getHeight(null));
-        int drawingX = 18 + ((220 - width) / 2);
-        int drawingY = 365 + ((130 - height) / 2);
-        g2d.drawImage(img, drawingX, drawingY, width, height, Color.BLACK, null);
-
-    }
-
+/*
++++rr * MegaMekLab - Copyright (C) 2008
```

Comment:
Probably not intended

Reply:
That would explain why gimp was not responding to my attempts to zoom and switch to the rectangle select tool.